### PR TITLE
aide 0.15.1 -> 0.18.8-f445264

### PR DESCRIPTION
**Pull request by Zopolis4**
### Run the following to get this pull request's changes locally for testing.
```bash
CREW_REPO=https://github.com/Zopolis4/chromebrew.git CREW_BRANCH=ultimatebattle crew update
```

### CODE REVIEW

File: manifest/x86_64/a/aide.filelist [hunk 1]
```diff
@@ -1,6 +1,3 @@
 /usr/local/bin/aide
-/usr/local/etc/aide.conf
-/usr/local/etc/doc/aide.db
```
Comment (uberhacker) [inline manifest/x86_64/a/aide.filelist:3]: What happened to the database and config?

Comment (Zopolis4) [reply manifest/x86_64/a/aide.filelist:3]: The config was actually a manpage, so shouldn't be installed like that, and the database was an empty file.

Comment (uberhacker) [reply manifest/x86_64/a/aide.filelist:3]: What?  This is what makes this thing work.

Comment (Zopolis4) [reply manifest/x86_64/a/aide.filelist:3]: `aide.conf.5` is a manual page, not a configuration file. The `doc/aide.conf` file that was present in `0.15.1` is no longer there.

File: packages/aide.rb [hunk 2]
```diff
@@ -1,54 +1,34 @@
...
-    puts
-    puts "The configuration file is located at #{CREW_PREFIX}/etc/aide.conf".lightblue
-    puts
-    puts 'To initialize the database, execute the following:'.lightblue
-    puts "mkdir #{CREW_PREFIX}/etc/src".lightblue
-    puts "cd #{CREW_PREFIX}/etc/doc".lightblue
-    puts 'aide -i'.lightblue
-    puts 'mv aide.db.new aide.db'.lightblue
-    puts
-  end
```
Comment (uberhacker) [inline packages/aide.rb:53]: Take a look at this postinstall message and see if you can get this working with your updates.

**Pull request closed without merging**